vignettes/tlf-kmplot.rmd

```diff
@@ -27,45 +27,45 @@ devtools::load_all()
 # Step 1: Read in data
 
 ```{r}
-adsl <- read_sas(file.path(path$adam, "adsl.sas7bdat"))
-adtte <- read_sas(file.path(path$adam, "adtte.sas7bdat")) 
+adsl <- read_xpt(file.path(path$adam, "adsl.xpt"))
+adtte <- read_xpt(file.path(path$adam, "adtte.xpt")) 
 ```
 
 ## Step 2: Data preparation for endpoint TTDE
 ```{r}
 anl <- adsl %>% 
   dplyr::filter(
-    ITTFL == "Y",
+    SAFFL == "Y",
```

Comment:
I feel `ITTFL` is proper for kmplot

Reply:
Thanks @elong0527 for your comment.
TTDE is for safety endpoints. Isn't SAFFL more appropriate for safety outputs?
I can change it back to ITTFL. Please let me know.

Reply:
I probably misunderstood the endpoint. If it is a safety endpoint then we shall use `SAFFL`. Thanks!